clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -252,6 +252,16 @@ offline_instru_t::append_pid(byte *buf_ptr, process_id_t pid)
     return sizeof(offline_entry_t);
 }
 
+int
+offline_instru_t::append_target_cache_line_size(byte *buf_ptr)
+{
+    offline_entry_t *entry = (offline_entry_t *)buf_ptr;
+    entry->extended.type = OFFLINE_TYPE_EXTENDED;
+    entry->extended.ext = OFFLINE_EXT_TYPE_TARGET_CACHE_LINE_SIZE;
+    entry->extended.valueA = proc_get_cache_line_size();
```

Comment:
assert that nothing is lost on truncation

Reply:
Done. Let me know if there's a better way than the current one.